gradle.properties

```diff
@@ -1 +1 @@
-version=0.0.25
+version=0.0.26
```

Comment:
I guess maybe we should bump it to `0.0.27` given we already have `0.0.26` published. We should also update the `CHANGELOG.md` file, adding

```
## [0.0.27] - 2020-12-18
- Improvements to the Gradle publishing process

## [0.0.26] - 2020-12-06
- Improvements to the Gradle publishing process
```